### DISCUSSION TITLE
Question about Default categories

### DISCUSSION THREAD

**Discussion by mehr32** [searxng/searxng#2938] [General]
Hello, I hope you are doing well.

I have a question regarding a section in the settings called "Default categories."
Please consider that in addition to the general category, I have also activated the "Images" category in this section.

Now, my question is how can I request a specific number of images?
For example, I want twenty images from the Images category to be included in the results on my own server by default.

Thank you for your time :) If you can't explain it in full detail, please give me a clue so I can do this.

**Comment by unixfox** [comment]
That's unfortunately not possible, some engine does not allow to restrict the amount of images that can be requested.

**Reply by mehr32** [reply]
Ummm, there is no way? Do you have any ideas?

**Reply by unixfox** [reply]
There is no support like this built in SearXNG. If you want to restrict the amount of images per page then you will need to only activate the engine that you want and either limit in python the number of results or find how to restrict the number of results on the specific engine.

Example code for the google images engine: https://github.com/searxng/searxng/blob/master/searx/engines/google_images.py
